### PR TITLE
Several character rendering erros: "z.u00A0B." instead of "z. B."

### PR DESCRIPTION
**Pull request by freshking**
## Description

Fix no-break space character. Should be `\U00A0` instead of `\u00A0`.

## Link to Jira

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8240

### CODE REVIEW

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings [hunk 1]
```diff
@@ -182,7 +182,7 @@
...
 
-"ExposureSubmission_Antigen_Profile_Legal_Text04" = "Auf Ihr persönliches Schnelltest-Profil haben zunächst nur Sie Zugriff. Nur wenn Sie den QR-Code des Schnelltest-Profils anderen Personen zeigen, können diese Zugriff auf die persönlichen Daten Ihres Schnelltest-Profils erhalten. Ein Zugriff auf andere Daten aus der App erfolgt dabei nicht (z.\u00A0B. Ihre Check-ins oder Angaben im Kontakt-Tagebuch).";
+"ExposureSubmission_Antigen_Profile_Legal_Text04" = "Auf Ihr persönliches Schnelltest-Profil haben zunächst nur Sie Zugriff. Nur wenn Sie den QR-Code des Schnelltest-Profils anderen Personen zeigen, können diese Zugriff auf die persönlichen Daten Ihres Schnelltest-Profils erhalten. Ein Zugriff auf andere Daten aus der App erfolgt dabei nicht (z.\U00A0B. Ihre Check-ins oder Angaben im Kontakt-Tagebuch).";
```
Comment (NataliaLemmerth) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings:185]:
```suggestion
"ExposureSubmission_Antigen_Profile_Legal_Text04" = "Auf Ihr persönliches Schnelltest-Profil haben zunächst nur Sie Zugriff. Nur wenn Sie den QR-Code des Schnelltest-Profils anderen Personen zeigen, können diese Zugriff auf die persönlichen Daten Ihres Schnelltest-Profils erhalten. Ein Zugriff auf andere Daten aus der App erfolgt dabei nicht (z.\U00A0B. Ihre Check-Ins oder Angaben im Kontakt-Tagebuch).";
```

**Review by NataliaLemmerth — DISMISSED** [review]
Please change 'Check-ins' to 'Check-Ins'

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Comment by Ein-Tim** [comment]
@NataliaLemmerth 

The capital I of "Check-Ins" contradicts these screens:

**Review by pascal-brause — APPROVED** [review]

**Comment by NataliaLemmerth** [comment]
> @NataliaLemmerth
> 
> The capital I of "Check-Ins" contradicts these screens:
> 
> 
> 
> 

Hi @Ein-Tim Thanks for noticing. We'll handle it in another PR. The correct spelling for German is Check-In.

**Review by ArturFriesen — APPROVED** [review]

**Pull request merged**